src/SSHLibrary/abstractclient.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, host, alias=None, port=22, timeout=3, newline='LF',
         self._shell = None
         self._started_commands = []
         self.client = self._get_client()
+        self.banner = None
```

Comment:
I don't thing `self.banner` is needed in general but if it is, it should be used just on the Python side.